docs/self-hosted-appcircle/installation-podman.md

```diff
@@ -0,0 +1,990 @@
+
+### Supported Linux Distributions
+
+Self-hosted Appcircle server utilizing Podman, can only be installed on Linux operating system.
+
+- CentOS 8 or later
+- RHEL 8 or later
+
+:::info
+
+We are working on Ubuntu and Debian support. It will be available soon.
+
+:::
+
+### Hardware Requirements
+
+Minimum hardware requirements for self-hosted Appcircle can be:
+
+- 100GB or more free disk space
+- 4 or more cores CPU
+- 8 or more gigabytes (GB) RAM
+
+:point_up: These hardware specs are minimum requirements for basic execution and it can be used only for quick evaluation or development purposes.
+
+:::caution
+
+CPU architecture must be AMD or Intel 64-bit arch (`x86_64`).
+
+:::
+
+:::info
+
+If you have enough RAM and a recent CPU, performance of Appcircle server can be limited by hard drive seek times. So, having a fast drive like a solid state drive (SSD) improves runtime.
+
+:::
+
+Higher numbers will be better especially for increased number of users.
+
+For an enterprise installation, **minimum** hardware requirements are
+
+- 500GB SSD
+- 8 CPU
+- 16GB RAM
+
+For production environments, **recommended** hardware requirements are
+
+- 1TB SSD
+- 32 CPU
+- 64GB RAM
+
+:::caution
+
+#### Swap
+
+Using **swap** file lets self-hosted Appcircle server exceed the size of available physical memory. On memory pressure system will go on its operations with minimal degradation, when SSD used as hardware.
+
+So, we are recommending **swap** file usage on Linux.
+
+Its size should be minimum half of the RAM size. For example if you have 64 GB RAM, then you should choose minimum 32 GB swap file size. 64 GB will be better.
+
+#### Swappiness
+
+The `swappiness` parameter configures how often your system swaps data out of RAM to the swap space. So, it's an important setting for swap usage and affects performance.
+
+`10` is recommended value for `swappiness`.
+
+:books: For details on how to configure **swap** and `swappiness` parameter, follow guide in [here](https://www.digitalocean.com/community/tutorials/how-to-add-swap-space-on-ubuntu-22-04).
+
+:::
+
+### Podman Requirements
+
+The Appcircle server supports Podman as the container runtime. The minimum required version of Podman is 4.0.0 or higher.
+
+#### Podman Network Stack
+
+To ensure successful operation of the Appcircle server, it is required to switch from the CNI network stack to Netavark if Podman is bundled with CNI. You can see your network stack by running:
+
+```bash
+podman info | grep -i networkBackend
+```
+
+You can switch to Netavark network stack by simply installing Netavark and configure podman to use Netavark. 
+  
+```bash
+sudo dnf install -y netavark
+```  
+Once the installation is complete, please follow these steps to configure Podman:
+- Edit the /usr/share/containers/containers.conf file.
+- Add the following content to the [network] section:
+```bash
+network_backend="netavark"
+```
+- Save the file.
+- Reset Podman by running the command: `podman system reset`.
```

Comment:
I tested the command. It does not require `sudo`. But it resets everything.

```bash
[centos@centosstream8 ~]$ podman system reset
WARNING! This will remove:
        - all containers
        - all pods
        - all images
        - all networks
        - all build cache
        - all machines
        - all volumes
Are you sure you want to continue? [y/N] y
[centos@centosstream8 ~]$
```
So, if needed it should be done before using system in order to prevent data loss.

@safderun you should add this item as a backlog issue. :memo:

Reply:
But they can't use the system before they reset podman with new netavark network stack. Should we add a warning about that?

Reply:
OK, I saw the note below:

> If you skip the step about podman network stack above, you will encounter network related issues. Please make sure you have completed this step.

So, we're informing the user about the "must" step before usage. :+1: